src/renderers/dom/shared/ReactDOMComponentTree.js

```diff
@@ -47,7 +44,7 @@ function shouldPrecacheNode(node, nodeID) {
  */
 function getRenderedHostOrTextFromComponent(component) {
   var rendered;
-  while ((rendered = component._renderedComponent)) {
+  while (rendered = component._renderedComponent) {
```

Comment:
I can see how the over-parenthesization is providing value here. Indicates that I know what I'm doing.